test/suite/test_hs18.py

```diff
@@ -36,6 +37,11 @@ def timestamp_str(t):
 class test_hs18(wttest.WiredTigerTestCase):
     conn_config = 'cache_size=5MB,eviction=(threads_max=1)'
     session_config = 'isolation=snapshot'
+    key_format_values = [
+        # ('column', dict(key_format='r')),
```

Comment:
We haven't really changed this test yet, though we have disabled column store scenario, keys used are still string/row-store. Please change the keys to be scenario dependant

Reply:
I think we should remove the added make scenarios from this PR and deal with that in the individual tickets.

Reply:
Cool, lets do that, lets remove the scenarios from the tests that are still disabled.